### ISSUE TITLE
Chan_SCCP Manager add SCCP extension throws error with  "Fields are Empty"

### ISSUE THREAD

**Issue by rickwaynedad** [marcelloceschia/chan-sccp-b#608]
When attempting to add a new extension to SCCP manager, it fails with an error message of "Fields are Empty".

**Reproduction Steps:**

1. click on SCCP Connectivity
2. Click on add extension
3. add relevant fields and click submit
4. The extension is created but shows as "virtual" extension and wont accept calls from the outside

**chan-sccp version:** 14.5.0.4)

**asterisk version:** 16.26.0

**Expected behavior:**

extension is created and shows as SCCP

**Observed behavior:**

- Problem started happening recently, didn't happen in an older version of Chan-SCCP: [Yes/No] Yes
- Problem can be reliably reproduced, doesn't happen randomly: [Yes/No] Yes
- Platform: FreePBX 16.0.40.7
  - [CPU]
  - [32/64 Bit]
  - [Big/Small Endian]
  - [Operating System] 12.7.8-2204-1.sng7
  - [Operating System Version]
- Direct Contact information: [You can contact me here] rickwaynedad@gmail.com
- Log Files (attach .txt / [pastbin](http://pastebin.com/) / [gist](http://gist.github.com/)): [url]
- Comments: [Other Information]

**BTW: Consider making a donation via paypal:**
- https://www.paypal.com/cgi-bin/webscr?&cmd=_donations&business=chan.sccp.b.pp%40gmail.com

**Comment by antitrustx** [comment]
It happens when incorrect installation was happened
i got this issue.
to fix it: 
make mysql backup
-  drop tables , with alter table buttonconfig drop foreign key buttonconfig_ibfk_1;
                                                                    drop view sccpdeviceconfig;
                                                                    drop view sccplineconfig;
                                                                    drop table sccpbuttonconfig;
                                                                    drop table sccpdevice;
                                                                    drop table sccpdevmodel;
                                                                    drop table sccpline;
                                                                    drop table sccpsettings;
                                                                    drop table sccpuser;
  - Goto Freepbx admin - module admin and reinstall module 
  -  Import script sql like installation requires  mysql -u root asterisk < mysql-v5_enum.sql
  - in mysql make SET foreign_key_checks = 0; it need to convert charset
  - then make fix  with all necessary tables(sccpbuttonconfig, sccpuser, sccpsettingsm, sccpline, sccpdevmodelwith ALTER TABLE <SCCPTABLESHERE> CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;
  - and get back SET foreign_key_checks = 1;

After that everything is just fine.

**Comment by TexsModShop** [comment]
Yeah getting the same issue here @antitrustx's fix seems to not work anymore/my setup is bugged
